app/controllers/wyeworkers_controller.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class WyeworkersController < ApplicationController
+  include RestJsonUtils
+
+  before_action :set_wyeworker, only: %i[show update destroy]
+  before_action :explain_no_links_editing, only: %i[create update]
+
+  EXPOSED_PLAIN_ATTRIBUTES = %w[name id].freeze
+  WyeworkerKind = Wyeworker
+
+  def wyeworker_to_rep(wyeworker)
+    {
+      **wyeworker.slice(*EXPOSED_PLAIN_ATTRIBUTES),
+      initiatives: wyeworker.initiatives.map { |i| initiative_to_url(i) }
+    }
+  end
+
+  def rep_to_wyeworker(rep)
+    self.class::WyeworkerKind.new(rep)
+  end
```

Comment:
I'm guessing this is some kind of serialization/deserialization but I don't understand why this would be necessary